src/manager.c

```diff
@@ -194,3 +196,153 @@ manager_find_compartment_by_ddc(void* __capability ddc)
 {
     return loaded_comp; // TODO
 }
+
+void
+toml_parse_error(char* error_msg, char* errbuf)
+{
+    printf("%s: %s\n", error_msg, errbuf);
+    exit(1);
+}
+
+char
+get_size_of_str_type(char* str_type)
+{
+    if (!strcmp(str_type, "int"))
```

Comment:
I think it's generally clearer and more idiomatic to say `if (strcmp(...) == 0)`.

Reply:
I did a cursory search, and couldn't find either way more prominent than the other. If you don't have strong feelings, I'd like to keep it this way, but I am open to changing if you prefer.